### DISCUSSION TITLE
Can you do double screening in ASReview?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#550] [Q&A] [answered]
Is it possible to work with two colleagues simultaneously on the same project for example?

**Comment by J535D165** [comment, marked as answer]
At the moment of writing, ASReview version 0.16, ASReview doesn't support multiple users to work on the same project. 

Users can create two projects and upload the same dataset to both projects. In that case, the users train the two models independently. The results can be compared after exporting the dataset with label decisions from both projects. You can compare the results in Excel for example. 

Multiple users training the same model is not supported at the moment.

**Reply by SagevdBrand** [reply]
In today's advanced workshop for ASReview this question came up again and, @boer0107, our information specialists also suggested the following useful advice. You could start with screening a small part (2% for example) of your dataset randomly with multiple screeners, to crystallize the specific articles you are looking for. Using that information you can start reviewing in ASReview using the two projects as described in the answer above. 

Another handy suggestion is to add a unique identifier for each record, before you upload them to ASReview. I believe reference managers such as Endnote can do this for you. Then, after multiple screeners have screened the dataset, you can use the unique identifiers to easily compare the decisions.
